### DISCUSSION TITLE
How do I change the focus, so that its not always portrait mode with blurry background?

### DISCUSSION THREAD

**Discussion by haab1001** [lllyasviel/Fooocus#1257] [Q&A]
How do I change the focus, so that its not always portrait mode with blurry background?

**Comment by tidianeb5** [comment]
i want to know too ?

**Reply by tidianeb5** [reply]
@haab1001  did you find a solution ?

**Reply by haab1001** [reply]
No solution yet.. :/

**Comment by Hypersonic-Popcorn** [comment]
Turn on Advanced mode and you should have a lot of resolutions to choose from, including square and landscape. Also, you might want to look at the style tab. The default ones make really nice pictures but changing them may help you get what you want.

**Reply by tidianeb5** [reply]
thanks
